resources/config.rb

```diff
@@ -23,14 +23,11 @@
          required: true,
          coerce: proc { |m| m.is_a?(Hash) ? Mash.new(m) : m }
 property :service_group, String, name_property: true, desired_state: false
-property :peer, String, desired_state: false
-property :org, String, desired_state: false
-property :ring, String, desired_state: false
-property :api_host, String, default: '127.0.0.1', desired_state: false
-property :api_port, Integer, default: 9631, desired_state: false
+property :remote_sup, String, desired_state: false
+property :user, String, desired_state: false
 
 load_current_value do
-  uri = URI("http://#{api_host}:#{api_port}/census")
+  uri = URI("http://#{remote_sup}/census")
```

Comment:
We used to have a default value with the old parameters - it'd be good to make sure this has a sensible default too.

Reply:
I had left no default so that if nothing was provided, hab cli would take the default action (which is to talk to localhost:9631). I can definitely update this if we want to make sure Chef always sets a default external to the Hab CLI default.

Reply:
I'd prefer to have a default set in the code as we did before for clarity :)

Reply:
Will do :)

Reply:
Done